datasets/duurzaamheid/energielabel/dataset.json

```
@@ -100,7 +100,7 @@
             "description": "Huisnummer van het verblijfsobject"
           },
           "huisletter": {
-            "type": "integer",
+            "type": "varchar",
```

Er is geen `varchar` in Amsterdam Schema.

```suggestion
            "type": "string",
```